Sources/CNIOLinux/include/ifaddrs-android.h

```diff
@@ -48,10 +48,12 @@ struct ifaddrs {
   unsigned int ifa_flags;
   struct sockaddr* ifa_addr;
   struct sockaddr* ifa_netmask;
-  union {
-      struct sockaddr *ifu_broadaddr;
-      struct sockaddr *ifu_dstaddr;
-  } ifa_ifu;
+union {
+struct sockaddr *ifu_broadaddr;
+struct sockaddr *ifu_dstaddr;
+} ifa_ifu;
+  /* Real ifaddrs has broadcast, point to point and data members.
+   * We don't need them (yet?). */
```

Comment:
For future reviewers: this change was introduced upstream, we're not worrying about it here.